pkg/terraform/install.go

```diff
@@ -28,6 +33,25 @@ func (m *Mixin) Install() error {
 		step.Flags = append(step.Flags, builder.NewFlag("input=false"))
 	}
 
+	// Only create a tf var file for install
+	if !step.DisableVarFile && action.Name == "install" {
+		vf, err := m.FileSystem.Create(path.Join(m.WorkingDir, m.TerraformVarsFilename))
+		if err != nil {
+			return err
+		}
+		defer vf.Close()
+		vbs, err := json.Marshal(step.Vars)
+		if err != nil {
+			return err
+		}
+		_, err = vf.Write(vbs)
+		if err != nil {
+			return err
+		}
+		if m.Debug {
+			fmt.Printf("TF var file created:\n%s\n", string(vbs))
```

Comment:
It looks like the [pattern in Porter](https://github.com/getporter/porter/blob/main/pkg/exec/builder/execute.go#L137-L139) is to print debug messages to the Err stream, in this case `m.Err`.  Can we update this to do the same?

Reply:
I've been away from the internet for bit.  :)  I'll get this in tomorrow.